adobe_vipm/flows/sync/agreement.py

```diff
@@ -529,6 +522,22 @@ def _add_3yc_fulfillment_params(
 
         return new_parameters
 
+    def _update_3yc_ordering_params(self, commitment_info: dict, parameters: dict):
+        new_parameters = copy.deepcopy(parameters)
```

Comment:
Why do we need a deep copy here? 

Same for this line -> https://github.com/softwareone-platform/swo-adobe-vipm-extension/pull/676/changes#diff-894c21200963e2fba6d47b6d740ed2606c9e431eee68b90fa6e49c0295c2772dR477

which is actually an empty dictionary all the time.